# With default build.xml, "jsa" plugin fails to build:
# tuxguitar-1.4-src/TuxGuitar-jsa/src/org/herac/tuxguitar/player/impl/jsa/assistant/SBAssistant.java:13: 
# error: package org.herac.tuxguitar.editor.action does not exist
# [javac] import org.herac.tuxguitar.editor.action.TGActionProcessor


diff -ur tuxguitar-1.4-src.org/TuxGuitar-jsa/build.xml tuxguitar-1.4-src/TuxGuitar-jsa/build.xml
--- tuxguitar-1.4-src.org/TuxGuitar-jsa/build.xml
+++ tuxguitar-1.4-src/TuxGuitar-jsa/build.xml
@@ -14,6 +14,7 @@
 	<path id="class.path">
 		<pathelement location="${path.tuxguitar-lib}"/>
 		<pathelement location="${path.tuxguitar-gm-utils}"/>
+		<pathelement location="${path.tuxguitar-editor-utils}"/>
 		<pathelement location="${path.tuxguitar}"/>
 		<pathelement location="${path.tuxguitar-ui-toolkit}"/>
 	</path >
